kura/org.eclipse.kura.core.status/src/main/java/org/eclipse/kura/core/status/runnables/BlinkStatusRunnable.java

```diff
@@ -11,29 +11,36 @@
  *******************************************************************************/
 package org.eclipse.kura.core.status.runnables;
 
-import org.eclipse.kura.KuraException;
 import org.eclipse.kura.core.status.LedManager;
 import org.eclipse.kura.status.CloudConnectionStatusEnum;
 
-public class BlinkStatusRunnable implements Runnable {
+public class BlinkStatusRunnable implements StatusRunnable {
 
     private final LedManager ledManager;
 
+    private boolean enabled;
+
     public BlinkStatusRunnable(LedManager ledManager) {
         this.ledManager = ledManager;
+        this.enabled = true;
     }
 
     @Override
     public void run() {
-        while (true) {
+        while (this.enabled) {
             try {
                 this.ledManager.writeLed(true);
                 Thread.sleep(CloudConnectionStatusEnum.SLOW_BLINKING_ON_TIME);
                 this.ledManager.writeLed(false);
                 Thread.sleep(CloudConnectionStatusEnum.SLOW_BLINKING_OFF_TIME);
-            } catch (InterruptedException | KuraException ex) {
+            } catch (Exception ex) {
                 break;
```

Comment:
Is it possible to catch a more specific exception?

Reply:
Change made